src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/SinkIslandCheck.java

```diff
@@ -28,13 +30,19 @@
  * @author matthieun
  * @author cuthbertm
  * @author gpogulsky
+ * @author savannahostrowski
  */
 public class SinkIslandCheck extends BaseCheck<Long>
 {
-    public static final float LOAD_FACTOR = 0.8f;
-    public static final long TREE_SIZE_DEFAULT = 50;
-    private static final List<String> FALLBACK_INSTRUCTIONS = Arrays
-            .asList("Road is impossible to get out of.");
+    private static final float LOAD_FACTOR = 0.8f;
+    private static final long TREE_SIZE_DEFAULT = 50;
+    private static final List<String> FALLBACK_INSTRUCTIONS = Collections
+            .singletonList("Road is impossible to get out of.");
+    private static final String MOTORCYCLE_PARKING_AMENITY = "MOTORCYCLE_PARKING";
+    private static final String PARKING_ENTRANCE_AMENITY = "PARKING_ENTRANCE";
+    private static final List<String> amenityValuesToExclude = Arrays.asList(
+            AmenityTag.PARKING.toString(), AmenityTag.PARKING_SPACE.toString(),
+            MOTORCYCLE_PARKING_AMENITY, PARKING_ENTRANCE_AMENITY);
```

Comment:
Please consider opening an issue on atlas project to add these tags to `AmenityTag`.

Reply:
Okay, so I'll leave that as is for now, and open that issue.